tools/pkg/repo/create.py

```diff
@@ -530,7 +532,14 @@ def _create_repo_file(create_repo_path, url_suffix):
 
     _create_repo_file(repo_file_path, f"minor/{salt_version}")
 
-    if not nightly_build_from:
+    if nightly_build_from:
+        nightly_latest_repo_file_path = create_repo_path.parent / "nightly_latest.repo"
+        _create_repo_file(nightly_latest_repo_file_path, "minor/nightly", "latest")
+
+        nightly_link = create_repo_path.parent / "minor/nightly"
```

Comment:
I have replied to this change in https://github.com/saltstack/salt/pull/64928#pullrequestreview-1566716388

Reply:
adjusted to `minor/latest`